### DISCUSSION TITLE
Is there any plugin handles `path` smartly?

### DISCUSSION THREAD

**Discussion by otakutyrant** [neovim/neovim#24261] [Q&A] [answered]
I am reading Practical Vim and it mentions that you can jump to a file under the cursor with `gf`. When you press `gf`, Vim searches the file in its `path`. The book recommends `vim-bundler` this plugin to handle path smartly when you are editing Ruby files.

However. No matter whether I open a Python file with Neovim in a virtual environment, the `path` is always `./usr/include,,`. So I look for any plugin can handles `path` for any programming languages smartly. I searched `path` in neovimcraft and no results.

**Comment by yutkat** [comment, marked as answer]
https://github.com/hrsh7th/nvim-gtd

**Reply by otakutyrant** [reply]
Thank you!

**Comment by otakutyrant** [comment]
Today I tried to use `ctrl+]` without any ctages generated and I jump to the source code of implementation successfully. It seems that I do not need any plugin to handle `path`.
